usr/share/rear/rescue/default/500_ssh.sh

```diff
@@ -3,164 +3,97 @@
 # This file is part of Relax-and-Recover, licensed under the GNU General
 # Public License. Refer to the included COPYING for full text of license.
 
-if has_binary ssh || has_binary sshd; then
+has_binary ssh || has_binary sshd || return
 
-    # assume that we have openssh with configs in /etc/ssh
+# assume that we have openssh with configs in /etc/ssh
 
-    # The funny [] around a letter makes shopt -s nullglob remove this file from the list if it does not exist,
-    # files without a [] are mandatory.
+# The funny [] around a letter makes shopt -s nullglob remove this file from the list if it does not exist,
+# files without a [] are mandatory.
 
-    COPY_AS_IS=( "${COPY_AS_IS[@]}" /etc/ssh* /root/.ssh/[a]uthorized_keys /root/.shos[t]s )
-    PROGS=(
-        "${PROGS[@]}"
-        ssh sshd scp sftp ssh-agent
-        $(
-            read subsys sftp original_file junk < <( grep sftp /etc/sshd_co[n]fig /etc/ssh/sshd_co[n]fig /etc/openssh/sshd_co[n]fig /etc/centrifydc/ssh/sshd_co[n]fig 2>/dev/null )
-            echo $original_file
-        )
+COPY_AS_IS=( "${COPY_AS_IS[@]}" /etc/ssh* /root/.ssh/[a]uthorized_keys /root/.shos[t]s )
```

Comment:
Please look at https://linux.die.net/man/1/ssh under Files. There are many more files in the `/root/.ssh` than those that you copy.

I don't agree with this change in ReaR behavior. So far we copy all files. IMHO this should stay.